macros/g4simulations/G4_CEmc_EIC.C

```diff
@@ -269,6 +269,11 @@ void CEMC_Clusters(int verbosity = 0) {
     cemc_clusterbuilder->Detector("CEMC");
     cemc_clusterbuilder->Verbosity(verbosity);
 
+    ClusterBuilder->set_threshold_energy(0.030); // This threshold should be the same as in CEMCprof_Thresh**.root file below
```

Comment:
Is this addition supposedly be added in line 264, and with variable 'cemc_clusterbuilder' instead of 'ClusterBuilder'? 

Currently it cause errors in the default EIC macro test: 
https://web.racf.bnl.gov/jenkins-sphenix/job/sPHENIX/job/test-default-pipeline/5444/console
Other test is on going but looks OK.